exiftool.go

```diff
@@ -155,17 +170,21 @@ func (e *Exiftool) ExtractMetadata(files ...string) []FileMetadata {
 			continue
 		}
 
+		e.log("sending to exiftool STDIN:")
 		for _, curA := range extractArgs {
+			e.logf("\t%s\n", curA)
 			if _, err := fmt.Fprintln(e.stdin, curA); err != nil {
 				fms[i].Err = err
 				continue
 			}
 		}
 
+		e.logf("\t'%s'\n", f)
 		if _, err := fmt.Fprintln(e.stdin, f); err != nil {
 			fms[i].Err = err
 			continue
 		}
+		e.logf("\t%s\n", executeArg)
 		if _, err := fmt.Fprintln(e.stdin, executeArg); err != nil {
 			fms[i].Err = err
 			continue
```

Comment:
You make me realize how ugly this portion of code is !
I'll refactor it :)

Reply:
I frequently think that about code my past self has written. It's gotta start somewhere though :)